src/ert/_c_wrappers/enkf/_deprecation_migration_suggester.py

```diff
@@ -154,5 +155,22 @@ def add_suggestion(kw, suggestion):
             " be selected on request from the GUI, and there is also an alternative"
             " view accessible by using ert viz in the commandline.",
         )
+        add_suggestion(
+            "UPDATE_PATH",
+            "The UPDATE_PATH keyword has been removed and no longer has any effect. "
+            "It can be safely removed. It has been used in the path to set different "
```

Comment:
Let's not have "It can be safely removed" as the user technically needs to understand whether they need this line.